packages/ds-medicare-gov/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cmsgov/ds-medicare-gov",
-  "version": "3.4.0",
+  "version": "3.4.0-beta.1",
```

Comment:
Instead of releasing a 3.4.0, we're going to be releasing a 3.4.0 beta that will live on until we've got the green light to release it next year.